frontend/src/Components/Navbar/Mobile_Navbar.jsx

```
@@ -4,64 +4,73 @@ import { GetterContext } from "../Context/NewContext";
+                let path = item.path;
+                if (
+                  path == "/dashboard" &&
+                  myRole &&
+                  (myRole?.power || 0) == 0
+                ) {
+                  path =
+                    "/dashboard/departments/emp/" +
+                    encodeURIComponent(me?.id || "");
+                }
+                return (
+                  <li>
```

Biome (2.3.13)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)